javascript/javascript1/week4/creditCardNo/cc.js

```diff
@@ -0,0 +1,13 @@
+function formatCreditCardNumber(number) {
+    number = number.toString();
+    const gap_size = 4;
+    let result = "";
+    while (number.length > 0) // Loop through string
+    {
+        result = result + " " + number.substring(0, gap_size); // Insert space character
+        number = number.substring(gap_size);  // Trim String
+    }
+    return result;
+}
+const formattedCreditCardObject = formatCreditCardNumber(123456789);
```

Comment:
you call it `Object` here, but it's actually a string, be careful with your names, they will confuse you later.